src/components/CustomInfo/index.tsx

```diff
@@ -14,7 +14,7 @@ const CustomInfo = ({ title, skill, message }: Info) => {
       <Card className="d-flex align-items-center border-0 py-1 text-dark">
         <div>{title}</div>
         <div className="skill-container d-flex ">
-          <CustomSkill skill={skill} />
+          <CustomSkill skills={[]} />
```

Comment:
Why an empty array?

Reply:
Without it, the app will break in some components that are not written by me and which still use hard coded data. So, it's just a temporary solution. I didn't write the CustomSkill component, so in order form me to be able to use it, I needed to modify it but also apply this temporary solution in components that would otherwise break.